cmake/scripts/cc_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+REQUIRED_ENVS=("MESATEE_PROJECT_ROOT" "MESATEE_BUILD_ROOT" "CMAKE_C_COMPILER")
+for var in "${REQUIRED_ENVS[@]}"; do
+    [ -z "${!var}" ] && echo "Please set ${var}" && exit -1
+done
+
+exec "${CMAKE_C_COMPILER}" "$@" -fdebug-prefix-map=${MESATEE_PROJECT_ROOT}=/mesatee_src -fdebug-prefix-map=${MESATEE_BUILD_ROOT}=/mesatee_build
```

Comment:
Can you add a comment before this line saying that why do we need to replace `debug-prefix-map`.

Reply:
done